changelog/fragments/1684858962-Support-signed-UNENROLL-action-and-UNENROLL-forwarding-to-Endpoint.yaml

```
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: feature
+
+# Change summary; a 80ish characters long description of the change.
+summary: Support signed UNENROLL action and UNENROLL forwarding to Endpoint
```

Same I think we should make this clearer.

```suggestion
summary: Enable unenroll action to complete successfully when Endpoint protection is enabled.
```